docs/application/vstools/install.md

```diff
@@ -1,170 +1,102 @@
-# Installing Visual Studio Tools for Tizen
-
-You need the following components on top of Visual Studio to make Visual Studio Tools for Tizen work:
-
-- VSIX
-
-  Visual Studio extension for Tizen packaging
-
-- Tizen Baseline SDK
-
-  The SDK is shared with the Tizen Studio, and it supports tools (such as Certificate Manager, Device Manager, Emulator, SDB, and on-demand rpm) for developing Tizen .NET applications.
+# Introduction
 
+Visual Studio Tools for Tizen provides Tizen-specific tools to improve your productivity.
 
```

Comment:
Information Required: Could you provide some more details to give an overview.

Reply:
Visual Studio Tools for Tizen allow to create, build and debug Tizen applications using Visual Studio IDE. With Visual Studio Tools for Tizen you can:
* Create applications for TV, Wearable and Mobile Tizen variants
* Debug your application from within Visual Studio
* Profile performance and memory usage of your application